docs/pages/v3/compound-3.md

```diff
@@ -92,10 +92,31 @@ deployments:
       ARB: '0x912ce59144191c1204e64559fe8253a0e49e6548'
       COMP: '0x354A6dA3fcde098F8389cad84b0182725c6C91dE'
       GMX: '0xfc5A1A6EB076a2C7aD06eD22C90d7E710E35ad0a'
-      USDC: '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8'
+      USDC.e: '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8'
+      WBTC: '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f'
+      WETH: '0x82af49447d8a07e3bd95bd0d56f35241523fbab1'
+  Arbitrum - USDC Base:
+    tab_text: Arbitrum USDC (Native)
+    blockscan_origin: 'https://arbiscan.io/'
+    contracts:
+      cUSDCv3: '0x9c4ec768c28520B50860ea7a15bd7213a9fF58bf'
+      cUSDCv3 Implementation: '0xF25212E676D1F7F89Cd72fFEe66158f541246445'
+      cUSDCv3 Ext: '0x1B2E88cC7365d90e7E81392432482925BD8437E9'
```

Comment:
The extension delegate is actually a different address for this deployment (even though it's the same contract/byte code): [0x27C348936400791b7350d80Fb81Bc61Ad68dF4AE](https://arbiscan.io/address/0x27C348936400791b7350d80Fb81Bc61Ad68dF4AE)